### PR TITLE
Adds lumps of coal.

### PR DESCRIPTION
**Pull request by harmonyn**
Currently only spawns as one of the options for presents. More functionality suggestions are welcome.

### CODE REVIEW

**Comment by TehZombehz** [comment]
Bah, humbug.

**Comment by calzilla1** [comment]
Light it on fire and make it turn into a fire ball

**Comment by KorPhaeron** [comment]
Please leave a description of why you think this is worth adding

**Comment by bgobandit** [comment]
Because it is the archetypical joke gift for people on the naughty list.

File: code/game/objects/items/trash.dm [hunk 1]
```diff
@@ -67,3 +67,14 @@
...
 	return
+
+/obj/item/trash/coal
+	name = "lump of coal"
+	icon = 'icons/obj/mining.dmi'
+	icon_state = "slag"
+	desc = "Someone's gotten on the naughty list."
+
+/obj/item/trash/coal/burn()
+	visible_message("[src] fuses into a diamond! Someone wasn't so naughty after all...")
```
Comment (coiax) [inline code/game/objects/items/trash.dm]: power creep

Comment (bgobandit) [inline code/game/objects/items/trash.dm]: it's one diamond, from something with a 1/50 or thereabouts chance of being spawned during a christmas only event

Comment (coiax) [inline code/game/objects/items/trash.dm]: POWER CREEP

Comment (soulgamer) [inline code/game/objects/items/trash.dm]: Why no gun for Santa that gives all the bad boys and girls coal? Coal that just happens to be accelerated to lethal speeds is still coal after all!

Comment (jughu) [inline code/game/objects/items/trash.dm]: Krampus when

Comment (FantasticFwoosh) [inline code/game/objects/items/trash.dm]: You are aware you can artifically set objects on fire with a bonfire?

Collectively gather coal, burn into a diamond & then extinguish with a fire extinguisher.

**Pull request merged**